### PR TITLE
Dropdown: set up with jsx AND css

### PR DESCRIPTION
**Pull request by Nyakuma101**
File set up JSX and CSS
add the dropdown feature

### CODE REVIEW

**Comment by Nyakuma101** [comment]
dropdown

File: src/components/Dropdown/Dropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <label className={styles.label}>
+      <div className={styles.container}>
+        <div className={styles.text}>{text}</div>
+        <select className={styles.select}>
+          <option className={styles.NotSubmitted}>Not Submitted</option>
+          <option className={styles.Submitted}>Submitted</option>
+          <option className={styles.NoCallback}>No Callback</option>
+          <option className={styles.InterviewScheduled}>Interview Scheduled</option>
+          <option className={styles.Interviewing}>Interviewing</option>
+          <option className={styles.OfferReceived}>Offer Received</option>
```
Comment (FemmeQueenInSTEM) [inline src/components/Dropdown/Dropdown.jsx]: The options here are hard-coded for what we'll use them for, but ideally the Dropdown component accepts three things:

1. The options to display
2. The selected option
3. A method to set the selected options

That way the component can be used for _any_ scenario.

File: src/index.css [hunk 2]
```diff
@@ -17,6 +17,13 @@ body {
   --text-secondary-color: #9399a2;
   --cta-primary-color: #0095ff;
 
+  --status-grey: #cecece;
+  --status-yellow: #f2cf68;
+  --status-red: #fe655c;
+  --status-orange: #e5842b;
+  --status-cyan: #5be1f3;
+  --status-green: #74f669;
```
Comment (FemmeQueenInSTEM) [inline src/index.css:25]: Thanks for adding these!

File: src/components/Dropdown/Dropdown.jsx
Comment (Walflower) [inline src/components/Dropdown/Dropdown.jsx:1]: it giving the error and saying it needs a key=value pair for the child components

File: src/components/Dropdown/Dropdown.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+import styles from './Dropdown.module.css';
+
+export default function Dropdown({ text }) {
+  const [val, setVal] = useState('');
+  console.log(val);
```
Comment (FemmeQueenInSTEM) [inline src/components/Dropdown/Dropdown.jsx]: Please remove `console.log`s before committing!

File: src/components/Dropdown/Dropdown.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  const status = ['Not Submitted', 'Submitted', 'No Callback', 'Interview Scheduled', 'Interviewing', 'Offer Received'];
```
Comment (FemmeQueenInSTEM) [inline src/components/Dropdown/Dropdown.jsx:6]: Can you move this outside the component? It's specific to one use case of the dropdown

Comment (Nyakuma101) [reply src/components/Dropdown/Dropdown.jsx:6]: move what outside of dropdown? can you be more specific?

**Review by Walflower — APPROVED** [review]
😀

**Pull request merged**